ucm2/MediaTek/mtk-rt5650/HiFi.conf

```diff
@@ -0,0 +1,94 @@
+# Use case configuration for mtk-rt5650
+
+SectionVerb {
+	# ALSA PCM
+	Value {
+		TQ "HiFi"
+	}
+
+	Include.e.File "/codecs/rt5645/EnableSeq.conf"
+
+	EnableSequence [
+		cset "name='O03 I05 Switch' 1"
+		cset "name='O04 I06 Switch' 1"
+		cset "name='O09 I17 Switch' 1"
+		cset "name='O10 I18 Switch' 1"
+
+		cset "name='ADC Capture Volume' 47"
```

Comment:
Remove volume controls and add PlaybackMixerElem / CaptureMixerElem  to the device definitions to allow the hardware volume control for pulseaudio/pipewire.

Reply:
Hello, is there any way to set a volume limit in UCM2?

The shown values are supposed to be maximum limits and there shouldn't be any problem for the capture volume, but the speaker volume gets really too high, producing a lot of clipping and probably even damaging the speakers on the machine.

Reply:
If the hardware can be damaged, it's probably the driver issue. The driver should not allow this. You can set the initial (safe) volume in UCM only at the moment. The limiting may be implemented using a special plugin in alsa-lib (but it's not available at the moment).